Refine proposal for discovery (#55)
w3c-ccg/traceability-interop: docs/discovery.md

HTTP GET https://platform.example/organization/123/did.json => didDocument.json
```

> 2. Review the `alsoKnownAs`, `assertionMethod` and `authentication` sections of the did document.

Reviewer: ```suggestion
2. Review the `alsoKnownAs`, `assertionMethod`, and `authentication` sections of the DID document.
```

#### `alsoKnownAs`

> This relationship is a statement that the subject of this identifier is also identified by one or more other identifiers.

See [alsoKnownAs](https://www.w3.org/TR/did-core/#dfn-alsoknownas).

This entry MUST be present.
This entry MUST have a first element that is the base URL for a VC-API that supports presentation exchange.

#### `assertionMethod`

The set of supported `assertionMethod` DID URLs for the organization.
See [assertionMethod](https://www.w3.org/TR/did-core/#assertion).

#### `authentication`

The set of supported `authentication` DID URLs for the organization.
See [authentication](https://www.w3.org/TR/did-core/#authentication).

> The `didDocument` MAY contain a `verificatonMethod` section, which MAY be used to support `did:web` based verification relationships.

Reviewer: ```suggestion
The `didDocument` MAY contain a `verificationMethod` section, which MAY be used to support `did:web` based verification relationships.
```

> The `didDocument` MUST NOT contain `verificatonMethod`'s where the controller is different from the DID Subject.

Reviewer: ```suggestion
The `didDocument` MUST NOT contain `verificationMethods` where the controller is different from the DID Subject.
```
Style choice -- either "`verificationMethod`s" or "`verificationMethods`" (that is, ``` `verificationMethod`s``` or ``` `verificationMethods` ```); I think the latter is better.

    "https://www.w3.org/ns/did/v1",
    {
      "@vocab": "https://www.w3.org/ns/did/#"
    }
  ],
  "id": "did:web:platform.example:organization:123",
  "alsoKnownAs": [
    "https://platform.example/organization/123",
    "did:key:z6MksSdhqJH3VvzcX8WP6VbdB85e6T7aaL5yLLYeJLJrto8V",
    "did:key:z82LkpR3sPw87xdgs72J3EzGXChciBmhV6ukkbeAGFtCpauXHiEwtM2tyDcphRnLmKsB1fi"
  ],
  "assertionMethod": [
    "did:key:z6MksSdhqJH3VvzcX8WP6VbdB85e6T7aaL5yLLYeJLJrto8V#z6MksSdhqJH3VvzcX8WP6VbdB85e6T7aaL5yLLYeJLJrto8V",
    "did:key:z82LkpR3sPw87xdgs72J3EzGXChciBmhV6ukkbeAGFtCpauXHiEwtM2tyDcphRnLmKsB1fi#z82LkpR3sPw87xdgs72J3EzGXChciBmhV6ukkbeAGFtCpauXHiEwtM2tyDcphRnLmKsB1fi"
  ],
   "authentication": [
    "did:key:z6MksSdhqJH3VvzcX8WP6VbdB85e6T7aaL5yLLYeJLJrto8V#z6MksSdhqJH3VvzcX8WP6VbdB85e6T7aaL5yLLYeJLJrto8V",
    "did:key:z82LkpR3sPw87xdgs72J3EzGXChciBmhV6ukkbeAGFtCpauXHiEwtM2tyDcphRnLmKsB1fi#z82LkpR3sPw87xdgs72J3EzGXChciBmhV6ukkbeAGFtCpauXHiEwtM2tyDcphRnLmKsB1fi"
  ],
  "services": [
    {
      "id": "did:web:platform.example:organization:123#traceability-api",
      "type": "TraceabilityAPI",
      "serviceEndpoint": "https://platform.example/organization/123"
    }
  ]
}
```

> In this example, the organization suports authentication and credential issuance with the same 2 keys, identified via the DID URLs in the relationships.

Reviewer: ```suggestion
In this example, the organization suports authentication and credential issuance with the same two keys, identified via the DID URLs in the relationships.
```
